### DISCUSSION TITLE
Getting Compilation error when install signal server

### DISCUSSION THREAD

**Discussion by ankitnetsol** [indrawp/Setup-Guide#150] [General]
Hello,

   Anyone, please suggest me a solution to solve the compilation error when installing the signal server. I attached a screenshot of the error that I am getting when installing a signal server.

![Screenshot from 2022-03-01 10-07-01](https://user-images.githubusercontent.com/61275695/156105715-df139c3a-5b61-47ee-8201-d0f416d418f8.png)

Thanks

**Comment by naive-guy** [comment]
Try compiling it with latest Java version. Works fine with Java 17.

**Reply by ankitnetsol** [reply]
And what was the mvn version?
